paddle/fluid/platform/cuda_enforce.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "paddle/fluid/platform/enforce.h"
+#ifdef PADDLE_WITH_CUDA
+#ifdef PADDLE_CUDA_KERNEL_CHECK
+
+#include <cuda.h>
+#include <stdio.h>
+
+#define __LEN_ERROR_MSG 128
+
+#define PADDLE_ENFORCE_CHECK_CUDA_KERNEL()                               \
+  do {                                                                   \
+    if (UNLIKELY(ifCudaFail())) {                                        \
+      char msg[__LEN_ERROR_MSG] = {0};                                   \
+      int line = 0;                                                      \
+      char file[__LEN_ERROR_MSG] = {0};                                  \
+      get_msg_from_cuda(msg, file, &line);                               \
+      char err_msg_temp[__LEN_ERROR_MSG + 32] = {};                      \
+      snprintf(err_msg_temp, __LEN_ERROR_MSG, "Error:%s", msg);          \
```

Comment:
insert blank between `Error` and `:`, and maybe we can use `InvalidArgumentError` directly

Reply:
done, thx.